frontend/routes/viewerCanvas/viewerCanvas.component.tsx

```diff
@@ -65,13 +66,9 @@ export class ViewerCanvas extends React.PureComponent<IProps, any> {
 	}
 
 	public renderPins(prev, curr) {
-		const { viewer } = this.props;
-
-		const toAdd = pinsDiff(curr, prev);
-		const toRemove = pinsDiff(prev, curr);
-
-		toRemove.forEach(viewer.removePin.bind(viewer));
-		toAdd.forEach(viewer.addPin.bind(viewer));
+		if (this.shouldBeVisible) {
```

Comment:
@sanmont3drepo I think you touched this part of the code. Make snre t's in the place that you envisaged

Reply:
Yeah, the "updatePins" props shouldnt exist.